kolibri/utils/cli.py

```diff
@@ -139,15 +138,17 @@ def get_cext_path(dist_path):
     elif 'macosx' in platform:
         platform = 'macosx'
     dirname = os.path.join(dirname, platform)
-
-    if not os.path.exists(os.path.realpath(str(dirname))):
-        logging.warning('No C Extensions available for this platform.\n')
-    else:
-        sys.path = sys.path + [os.path.realpath(str(dirname))]
+    sys.path = sys.path + [os.path.realpath(str(dirname))]
 
 
 # Add path for c extensions to sys.path
-get_cext_path(os.path.dirname(kolibri_dist.__file__))
+get_cext_path(sys.path[-1])
+try:
+    import cryptography
+except ImportError:
+    # Fallback
+    logging.warning('No C Extensions available for this platform.\n')
+    sys.path = sys.path[:-1]
```

Comment:
This wouldn't be necessary if the C compiled modules shared the same directory `kolibri/dist` as the other distributed modules. I think from this example that it's probably better to use the same target package directory, you will be able to remove a lot of code.